.github/workflows/test-ignore.yml

```diff
@@ -0,0 +1,53 @@
+name: CI-ScratchPad-PR
```

Comment:
Is this still needed or is it just for reference?

Reply:
that's just for reference but can be removed actually...